x-pack/test/alerting_api_integration/spaces_only/tests/alerting/builtin_alert_types/index_threshold/fields_endpoint.ts

```diff
@@ -139,6 +139,12 @@ export default function fieldsEndpointTests({ getService }: FtrProviderContext)
       expect(field.name).to.eql('updated_at');
       expect(field.type).to.eql('date');
     });
+
+    // TODO: the pattern '*a:b,c:d*' throws an exception in dev, but not ci!
```

Comment:
This is a bit weird.  Seems like FT is running ES in some mode where bad cluster names won't result in exceptions, but running `yarn es` will.  If you enter the pattern above when running in dev (yarn es, yarn start), you'll see the exception logged in the console, but you don't see it in in FT.